Demo/Supporting Files/ar.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,2 @@
+/* Localized versions of Info.plist keys */
```

Comment:
No need to review any of these `.lproj` files -- everything from here down is auto-generated.